package.json

```
@@ -0,0 +1,52 @@
+{
+  "name": "vue-money-amount",
+  "description": "Used to display a localized money amount in the user's preferred locale, or a custom locale",
+  "author": {
+    "name": "Solutions Kumojin Inc.",
+    "email": "contact@kumojin.com"
+  },
+  "bugs": {
+    "url": "https://github.com/kumojin/vue-money-amount/issues"
+  },
+  "homepage": "https://github.com/kumojin/vue-money-amount#readme",
+  "license": "MIT",
+  "version": "0.1.0",
```

```suggestion
  "version": "1.0.0",
```
J'aime pas faire des version 0.x.x